Automattic/newspack-ads
# fix(editor): ad unit block

Closes #5 and fixes an issue that is probably WP 5.8-related.

I did think about moving the select to the editor sidebar, to accurately represent the super-small ad units in the block, but super-small ad units seem too much of an edge case to accommodate in such a way.

## How to test

1. On `master`,
1. Ensure you're on WP 5.8
2. Create 100x10 and a 10x100 ad units
3. Insert three Ad Unit blocks somewhere - choose the two units from 👆 and a regular one
4. Observe that the select fills the block on the regular ad unit
5. Observe that the block with the 10x100 ad unit is squished, and the select is a couple pixels thin only
6. Switch to this branch, rebuild, observe issues gone
